src/m365/pp/commands/managementapp/managementapp-add.spec.ts

```diff
@@ -132,7 +132,7 @@ describe(commands.MANAGEMENTAPP_ADD, () => {
   });
 
   it('handles error when retrieving information about app through name failed', async () => {
-    sinon.stub(request, 'get').callsFake(_ => Promise.reject('An error has occurred'));
+    sinon.stub(request, 'get').callsFake(_ => { throw 'An error has occurred'; });
```

Comment:
Actually I thought we will change it on all places 😅. also here.. But that's ok. Let's leave it as is and I will resolve this one when merging. 👍